### ISSUE TITLE
Plugin to turn presentation into courseware

### ISSUE THREAD

**Issue by egrosclaude** [hakimel/reveal.js#1687]
Now what I'd really kill for is a "courseware mode" plugin to:
1- Split screen into 2/3 filled with slide, 1/3 filled with aside markdown text 
2- No clock or next slide block
3- No two windows as in notes plugin
4- Make reveal.js start by default in courseware mode
5- Hot key toggles between courseware mode and "normal" presentation mode

I think this may be not so far from what the "notes" plugin does - I have been tinkering around with the notes plugin and made a half-baked version of this but regretfully my JS skills are not enough for items 3 to 5.
I have already mentioned this, but I don't think my request was understood-- too wordy or loose focus I guess. So here it goes again please please please - This would be a dream tool for us teachers!

**Comment by Sonaryr** [comment]
I just made something that might help you:
https://github.com/Sonaryr/reveal.js-coursemod

**Comment by egrosclaude** [comment]
I... just... This is... WONDERFUL!!! THANK YOU!!! 


Only things left are:
6- Style the notes text
7- Set default mode for individual slides (like, say, the cover slide)
That, Sir, would be perfection.

This already fits my use case perfectly. No joking, this allows a different new level of teaching!!!
Thank you again!

**Comment by Sonaryr** [comment]
No problem :-)
I didn't style the notes on purpose leaving this for the creator of presentations. I will add a section to the README on how to do that.
The default mode per slide is a nice idea, I will try to add that later today or this week. Also another plan I have is to make it possible to make notes appear together with fragments. Is that something you might be interested in?

Please to make any suggestion for it to the repo for the plugin itself so this issue can be closed.

**Comment by egrosclaude** [comment]
I'd be absolutely interested in syncing notes to fragments. Moving to the plugin repo and closing.
